### ISSUE TITLE
Move Form types to GmailBundle

### ISSUE THREAD

**Issue by jonny-no1** [fourlabsldn/GmailDoctrineBundle#9]
Would it be a good idea to move the form types to GmailBundle, @galeaspablo?

**Comment by galeaspablo** [comment]
Yes. But because the form type depends on syncsettingclass, you would need to move that over there as well. So you would need to abstract syncsettingclass in the gmailbundle in the same way the other classes (email, label, history) are abstracted.
